zktest/src/archive/test2/config.properties

```diff
@@ -2414,6 +2414,7 @@ B85-ZK-3810.zul=A,E,Locale,WebSocket,i18n
 B85-ZK-3743.zul=A,E,Servlet3,AsyncContext,complete,CometServerPush
 B85-ZK-3804.zul=A,E,MeshWidget,Column,Width,Visible
 ##zats##B85-ZK-3866.zul=A,E,radio,doClick,evt,stop,popup
+##zats##B85-ZK-3871.zul=A,E,shadowElement,listbox,grid,nested

```

Comment:
There is no `B85-ZK-3871.zul`.

Add 3 files:

- B85-ZK-3871-if.zul
- B85-ZK-3871-chooseif.zul
- B85-ZK-3871-choose.zul (Please include this file)